migrate/migrations/000010_add-feeds-table.up.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE feeds (
+    id INT UNSIGNED AUTO_INCREMENT PRIMARY KEY,
+    created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+
+    post_type TINYINT UNSIGNED,
```

Comment:
Need for efficient queries when filtering feeds on post_type (feed vs inbox)

Reply:
Should we enforce the same constraints as the `posts` table for these columns?